configs/go.json

```diff
@@ -0,0 +1,192 @@
+{
+  "defaults": {
+    "files": [
+      ".github/workflows/automerge.yml", 
+      ".github/workflows/go-test.yml", 
+      ".github/workflows/go-check.yml", 
+      ".github/workflows/releaser.yml", 
+      ".github/workflows/release-check.yml", 
+      ".github/workflows/tagpush.yml"
+    ],
+    "deploy_versioning": true,
+    "deploy_go": true
```

Comment:
Why is this needed? Isn’t this obvious from the file name?

Reply:
Yes, we could. I set it up this way so other configurations can also opt-in to flagged sections of copy-workflow easily. For example, here's my `testing.json` file from `testing` branch -https://github.com/protocol/.github/commit/43476995428996a90ca95bf838f084ba1a710c68. As you can see, to mimic what `go.json` does, all I had to do was to copy the `defaults` from `go.json`.

If we wanted to remove the need for `deploy_go` flag in `go.json`, maybe we could add `source` field to the target object on which we operate in copy-workflow. Then we could change the if condition around the go section to `deploy_go == true || source == go.json`.

Reply:
Let's keep it the way it is. Having a separate `source` will probably be more confusing.